opm/output/eclipse/EclipseReader.cpp

```diff
@@ -74,40 +74,50 @@ namespace {
                                          + ", mismatched number of cells" );
         }
 
-        using ds = data::Solution::key;
         data::Solution sol;
-        sol.insert( ds::PRESSURE, double_vector( pres ));
-        sol.insert( ds::TEMP,     double_vector( temp ));
-        sol.insert( ds::SWAT,     double_vector( swat ));
-        sol.insert( ds::SGAS,     double_vector( sgas ));
 
-        const auto apply_pressure = [&]( double x ) {
-            return units.to_si( UnitSystem::measure::pressure, x );
-        };
+        {
+            const auto apply_pressure = [&]( double x ) {
+                return units.to_si( UnitSystem::measure::pressure, x );
+            };
 
-        const auto apply_temperature = [=]( double x ) {
-            return units.to_si( UnitSystem::measure::temperature, x );
-        };
+            const auto apply_temperature = [=]( double x ) {
```

Comment:
This probably need to capture by reference (well, copying unitsystem isn't the *worst* thing in the world, but it's not really accomplishing anything).